motion_tracker/data_setup/parse_imagenet_bb.py

```diff
@@ -97,7 +97,7 @@ def parse_bb(bounding_box):
     all_bboxes = (parse_file(bbox_dir, filename) for filename in bbox_xml_filenames)
     end_lst = list(itertools.chain(*all_bboxes))
 
-    cols = ['filename', 'width', 'height', 'name', 'subcategory', 'xmin', 'xmax',
-            'ymin', 'ymax']
+    cols = ['filename', 'width', 'height', 'name', 'subcategory', 'x0', 'x1',
+            'y0', 'y1']
     output_df = pd.DataFrame(data=end_lst, columns=cols)
```

Comment:
We discussed this, so I imagine this is good. I'll be sure to change it everywhere we need to before merging in.